src/components/Rockets.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Rocket from './Rocket';
+
+import { fetchRockets } from '../Redux/rockets/rockets';
+
+const Rockets = () => {
+  const rockets = useSelector((state) => state.rocketReducer);
+  const dispatch = useDispatch();
+  // useEffect(() => {
+  //   if (rockets.length === 0) dispatch(fetchRockets());
+  // }, []);
+
```

Comment:
- [ ] As mentioned earlier concerning leaving commented in our project this also applies.